packages/cli/src/commands/index.ts

```diff
@@ -139,6 +145,7 @@ export function registerScriptCommand(program: Command) {
       (opt: string, opts: string[]) => (opts ? [...opts, opt] : [opt]),
       Array<string>(),
     )
+    .option(...disableWatchFileOption)
```

Comment:
There's no need for a watch flag here, we can make it the default, since we already take a snapshot of the config before building.

Reply:
okay